JsBasics/Advances_function_usage/task3.js

```diff
@@ -0,0 +1,26 @@
+let divide = (numerator, denominator) => {
+    if (denominator === 0 || typeof numerator !== "number" || typeof denominator !== "number") {
+        return "some agrument is not Number";
```

Comment:
```suggestion
        throw new Error("some agrument is not Number");
```
А взагалі, коли denominator 0 - має бути інша помилка, окрема.